CHANGELOG.md

```diff
@@ -2,13 +2,18 @@
 
 ## Unreleased
 
+New error codes:
+* Introduce Y058: Use `Iterator` rather than `Generator` as the return value
+  for simple `__iter__` methods, and `AsyncIterator` rather than
+  `AsyncGenerator` as the return value for simple `__aiter__` methods.
+* Introduce Y059: `Generic[]` should always be the last base class, if it is
+  present in a bases tuple.
```

Comment:
Is "bases tuple" a common term? It seems a little confusing as most people likely don't think of it primarily as a tuple. 
```suggestion
  present in the bases of a class.
```

Reply:
Yeah, I was struggling a little with this -- we use it one or two times in the CPython docs, but it's maybe not common parlance. Your wording looks good!